lessons/0_workflow/workflow.html

```diff
@@ -0,0 +1,120 @@
+<html>
+
+<head>
+  <title>Javascript orientation session!</title>
+  <link rel="stylesheet" href="../../reveal.js/css/reveal.css" />
+  <link rel="stylesheet" href="../../reveal.js/css/theme/moon.css" />
+  <link rel="stylesheet" href="../../reveal.js/lib/css/zenburn.css" />
+  <link rel="JavaScript" href="../../reveal.js/plugin/markdown/marked.js" />
+  <link rel="JavaScript" href="../../reveal.js/plugin/markdown/markdown.js" />
+  <style>
+    .reveal pre {
+      margin: 0;
+    }
+
+    .monospace {
+      font-family: monospace !important;
+    }
+
+    .illustration {
+      border: 0px;
+    }
+
+    .reveal li {
+      margin-bottom: 0.5em;
+    }
+
+    .reveal pre {
+      width: 100%;
+    }
+
+    .reveal pre code {
+      cursor: pointer;
+    }
+
+    .reveal li pre {
+      width: fit-content;
+    }
+
+    .logo {
+      position: fixed;
+      /* Sit on top of the page content */
+      z-index: 2;
+      /* Specify a stack order in case you're using a different order for other elements */
+    }
+
+    .great {
+      color: green
+    }
+  </style>
+</head>
+
+<body>
+  <a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
+  <div class="reveal">
+    <div class="slides">
+      <section>
+        <h1>Workflow</h1>
+      </section>
+      <section>
+        <h3>Steps to upload your homework</h3>
```

Comment:
🚫 upload

Git is about tracking changes. Sometimes there homework might just be a series of changes in several files. I think we should not use the word upload in the way. 

I purpose "Steps to submit your homework"